sa/storage-authority.go

```diff
@@ -776,6 +776,20 @@ func (ssa *SQLStorageAuthority) CountCertificatesRange(start, end time.Time) (co
 	return count, err
 }
 
+// CountPendingAuthorizations returns the number of pending, unexpired
+// authorizations for the give registration.
+func (ssa *SQLStorageAuthority) CountPendingAuthorizations(regID int64) (count int, err error) {
+	err = ssa.dbMap.SelectOne(&count,
+		`SELECT count(1) FROM pendingAuthorizations
+		 WHERE registrationID = :regID AND
```

Comment:
There's a regID index on pendingAuthorizations. I can make that a (regID, expires) index: is that what you're asking here?